config/locales/en.yml

```diff
@@ -18,3 +18,4 @@
 #
 # To learn more, please read the Rails Internationalization guide
 # available at http://guides.rubyonrails.org/i18n.html.
+en:
```

Comment:
?

Reply:
Without it all `ActiveModel` validation exceptions will be masked by i18n exception that the file is invalid.
Adding `en:` to he file makes it valid and allows to receive original validation exceptions.

Reply:
Actually I've found that removing the file has the same effect.
Initially all lines in file are commented out `#...`.
So I'm removing it instead of adding `en:` to it...